ktor-client/ktor-client-tests/common/test/io/ktor/client/tests/HttpTimeoutTest.kt

```diff
@@ -213,11 +208,8 @@ class HttpTimeoutTest : ClientLoader() {
 
     @Test
     fun testGetRequestTimeoutWithSeparateReceive() = clientTests(except("Js"), retries = 5) {
-//      https://youtrack.jetbrains.com/issue/KTOR-7847/Investigate-Flaky-timeout-tests-on-linuxX64
-        if (PlatformUtils.IS_NATIVE) return@clientTests
```

Comment:
Tried to rerun tests several times on CI and they always were successful. So removed this `if`